src/core/core.scale.js

```diff
@@ -514,6 +514,10 @@ module.exports = Element.extend({
 
 	// Get the correct value. NaN bad inputs, If the value type is object get the x or y based on whether we are horizontal or not
 	getRightValue: function(rawValue) {
+		// Float-bar support. Check if array so be aware of min/max Y values.
+		if (helpers.isArray(rawValue)) {
+			rawValue = rawValue[1];
```

Comment:
Originally we have already smth like this when handling objects in this function with parameters x and y (core.scale.js). 

// If it is in fact an object, dive in one more level
		if (rawValue) {
			if (this.isHorizontal()) {
				if (rawValue.x !== undefined) {
					return this.getRightValue(rawValue.x);
				}
			} else if (rawValue.y !== undefined) {
				return this.getRightValue(rawValue.y);
			}
		}

So, my thought it would be appropriate to handle arrays in such way. Its much more better cause it safe to not brake anything in other parts of the code, doesnt slow thhe code and you dont need to make additional checking in controller if the value is array to pass one of the element. What do you think?

Reply:
Objects are a bit different because they represent values on **different scales**, so `getRightValue` returns the value for the associated scale. In our case, the array represents **the value for the scale** and thus should be returned as is IMO.

> Its much more better cause it safe to not brake anything in other parts of the code

It's not that safe because you return an arbitrary value (the max) so "the rest of the code" will ignore the min which I think could introduce more issues: for example if a scale wants to determine the min value to display, it might not work correctly since it will only consider the max part of the float bar.

Reply:
I agree with @simonbrunel here. Returning an array seems logically since there are two values to plot against the axis.

Reply:
done